examples/testdata/path_matcher/openapi_swagger.json

```diff
@@ -17,16 +17,28 @@
     "https"
   ],
   "paths": {
-    "/shelves/{shelf}": {
+    "/libraries/{library_id}/shelves/{shelfId}": {
```

Comment:
Can we have test case of different path but same type like mentioned in b/158262661?

Reply:
Sure. I added an example in the unit test `TestMakePathMatcherFilter` in `listener_generator_test.go`. Service config gets too complex with multiple routes here, so it's better in unit test.